docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -6,14 +6,165 @@ The parent key for all of the following parameters is `vcluster`.
 
 [horizontal]
 type:: string
-default:: `syn-vcluster`
+default:: `syn-${_instance}`
 
 The namespace in which to deploy this component.
```

Comment:
Would be nice to have explicit information on whether the component supports having multiple instances in the same namespace (from what I can see in the implementation, it almost does, the only thing that would need to be adjusted is to make sure that the Namespace object will be mapped to the same output file if multiple instances want to use the same namespace).

Reply:
Also, I learned this from component-thanos the hard way, another parameter like `createNamespace` is required. Otherwise, every component instance "fights" over who is maintaining the namespace object (because some instance-identifying annotations get added by ArgoCD).

Reply:
Thanks for your inputs 😅 It sounds a tad bastelig - is there no way to see from jsonnet if there are multiple instances?

Reply:
IMHO for this component, we can just require that each instance *must* be in its own namespace

Reply:
> is there no way to see from jsonnet if there are multiple instances?

Currently not, might be something to add in Commodore.